Solutions/archTIS/Package/mainTemplate.json

```diff
@@ -30,20 +30,20 @@
     },
     "workbook1-name": {
       "type": "string",
-      "defaultValue": "NucleusCyberProtect",
+      "defaultValue": "NCProtect",
       "minLength": 1,
       "metadata": {
         "description": "Name for the workbook"
       }
     }
   },
   "variables": {
-    "solutionId": "nucleuscyber.nc-protect-azure-sentinel-data-connector",
+    "solutionId": "archtis.nc-protect-azure-sentinel-data-connector",
```

Comment:
update the new offerId in the solution meta data as well to maintain consistency